R/dependent-packages.R

```diff
@@ -60,16 +61,25 @@ find_deps <- function(pkgs, available = utils::available.packages(),
   unique(c(if (include_pkgs) pkgs, top_flat, rec_flat))
 }
 
-standardise_dep <- function(x) {
-  if (identical(x, NA)) {
-    c("Depends", "Imports", "LinkingTo")
-  } else if (isTRUE(x)) {
-    c("Depends", "Imports", "LinkingTo", "Suggests")
-  } else if (identical(x, FALSE)) {
-    character(0)
-  } else if (is.character(x)) {
-    x
+dep_types_hard <- function() c("Depends", "Imports", "LinkingTo")
```

Comment:
Is there a reason this needs to be a function rather than just a constant?

Reply:
Just trying to avoid non-function code in packages in general. Got bitten many times. But yeah, it could be a constant this time.